<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FINTER</class>
 <widget class="QMainWindow" name="FINTER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>FINTER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modo de interpolacion:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modoInterpolacion">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>99</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="agregar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mostrarPasos">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar pasos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Polinomio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="polinomio">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>439</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="especializarPunto">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Esp Punto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cambiarModo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cambiar modo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finalizar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Finalizar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <zorder>agregar</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>modoInterpolacion</zorder>
   <zorder>quitar</zorder>
   <zorder>mostrarPasos</zorder>
   <zorder>label_3</zorder>
   <zorder>polinomio</zorder>
   <zorder>especializarPunto</zorder>
   <zorder>cambiarModo</zorder>
   <zorder>finalizar</zorder>
   <zorder>tableWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
